tox.ini

```diff
@@ -22,6 +26,7 @@ deps =
     dj40: Django==4.0.*
     dj41: Django==4.1.*
     dj42: Django==4.2.*
+    dj50: Django>=5.0a1,<5.1
```

Comment:
Would pin this to minimum django 5.0 release version

Reply:
Sure, now it’s out :)